server/database/db.js

```diff
@@ -0,0 +1,8 @@
+const mongoose = require( 'mongoose' );
+
+module.exports = {
+    connectTo: function ( database = 'sandbox', host = 'localhost' )
```

Comment:
Cute abstraction of the connect functionality. If this function used global variables it would be very easy to change DB  location and or name.